pom.xml

```diff
@@ -174,6 +174,10 @@
             <groupId>org.jberet</groupId>
             <artifactId>jberet-schedule-executor</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jberet</groupId>
+            <artifactId>jberet-schedule-timer</artifactId>
+        </dependency>
```

Comment:
`scheduleTimer` already has this dep in its pom.  If you add this dep here, it will be made available to all sub-projects. I think currenlty only `scheduleTimer` sub-project needs `jberet-schedule-timer`.

Reply:
It was required for the restAPI example as well for the feature to be enabled (otherwise frontend schedule form is disabled).
I guess I can move that dependency there?

Reply:
ok, then let's add this dep in the top pom.xml to make it available to all sub-projects. Can you remove this dependency from `scheduleTimer` since it is already declared in its parent pom?

Reply:
`scheduleExecutor` sample project uses `jberet-schedule-executor`, without using `jberet-schedule-timer`. So adding dependency of `jberet-schedule-timer` to the top level pom.xml will force it onto `scheduleExecutor` sample, which is not needed.  So I'll add `jberet-schedule-timer` dep to `restAPI` sample.